keras/layers/convolutional_recurrent.py

```diff
@@ -329,6 +331,7 @@ def __init__(self, filters,
 
         self.dropout = min(1., max(0., dropout))
         self.recurrent_dropout = min(1., max(0., recurrent_dropout))
+        self.state_spec = [InputSpec(ndim=4), InputSpec(ndim=4)]
```

Comment:
This could be made more specific: the channel dimension can be specified. This would improve user-facing error messages.